packages/playground/components/src/FilePickerTree/index.tsx

```diff
@@ -1462,8 +1512,8 @@ const NodeRow: React.FC<{
 		if (isRenaming) {
 			setRenameValue(node.name);
 			renameHandledRef.current = false;
-			if (typeof window !== 'undefined' && window.requestAnimationFrame) {
-				window.requestAnimationFrame(() => {
+			if (typeof window !== 'undefined' && requestAnimationFrame) {
```

Comment:
The previous version of this code which explicitly referenced the `window` property made more sense to me. But mostly, this change made me pause and wonder:
Is there a reason for this change that I'm missing?

🤔 If we're trying to reduce references to `window` for some reason, the following would go further:
```suggestion
			if (typeof requestAnimationFrame === 'function') {
```

Reply:
Yeah good note, let's explore restoring that